internal/bundler/snapshots/snapshots_ts.txt

```diff
@@ -305,6 +305,12 @@ var Foo;(function(e){let a;(function(p){foo(e,p)})(a=e.Bar||(e.Bar={}))})(Foo||(
 ---------- /b.js ----------
 export var Foo;(function(e){let a;(function(p){foo(e,p)})(a=e.Bar||(e.Bar={}))})(Foo||(Foo={}));
 
+================================================================================
+TestTSValidConstEnum
+---------- /out.js ----------
+// /entry.ts
+let usage = bar(1, 2, 3, "stringsWorkToo", Foo.E);
```

Comment:
The unknown member of `Foo` is still emitted. I am not sure how to proceed here - whether to leave the current behavior (emit only a warning) or emit an error, as this most likely will never work

Suggestions are welcome